docs/_running_external.rst

```diff
@@ -0,0 +1,223 @@
+Workflow execution against an external Galaxy
+===============================================
+
+The first requirement for executing workflows on an external Galaxy server is
+a user account for that server. If you don't already have one, usegalaxy.org,
+usegalaxy.eu and usegalaxy.org.au all provide free accounts which can be used
+for this tutorial.
+
+Assuming you have selected a server for this tutorial and have an account, you
+need to retrieve the API key associated with that account. This can be found at
+{server_url}/user/api_key, or by going to the User dropdown menu, selecting
+Preferences and then clicking on Manage API key.
+
+Now you can run the workflow with:
+
+::
+
+    $ planemo run wf3-shed-tools.ga wf3-shed-tools-job.yml --engine external --url SERVER_URL --api_key YOUR_API_KEY
+
+
+If you want to set the name of the history in which the workflow executes, add
+``--history_name NAME`` to the command. You should be able to see the workflow
+executing in the web browser, if you navigate to the 'List all histories' view. 
+
+Typing ``--engine external --url SERVER_URL --api_key YOUR_API_KEY`` each time
+you want to execute a workflow is a bit annoying. Fortunately, Planemo provides
+the option to create 'profiles' which save this information for you. To create
+a new profile called ``tutorial_profile``, you can run a command like the
+following:
+
+::
+
+    $ planemo profile_create tutorial_profile --galaxy_url SERVER_URL --galaxy_user_key YOUR_API_KEY --engine external_galaxy
+
+
+
+This allows creation of multiple profiles (e.g. for different Galaxy servers).
+A list of all created profiles is provided by the ``profile_list`` subcommand:
+
+::
+
+    $ planemo profile_list
+    Looking for profiles...
+    tutorial_profile
+    usegalaxy-eu
+    usegalaxy-org
+    3 configured profiles are available.
+
+
+Once the new ``tutorial_profile`` is created, a workflow can be executed with:
+
+::
+
+    $ planemo run wf3-shed-tools.ga wf3-shed-tools-job.yml --profile tutorial_profile
```

Comment:
the next logical step would be to define a default_profile ... a profile that is used when no param is used and not profile. In the profile_list command, it could be marked with a * - pretty much like git branches. If you like this idea, could you please create an issue for it.